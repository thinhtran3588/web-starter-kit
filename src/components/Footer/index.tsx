import React from 'react';
import Link from 'next/link';
import { Typography } from '@material-ui/core';
import { useStyles } from './styles';

export const Footer = (): JSX.Element => {
  const classes = useStyles();

  return (
    <footer className={classes.root}>
      <Typography variant='body1'>
        &copy;{' '}
        <Link href='/'>
          <a>Thinh Tran</a>
        </Link>
        &nbsp;2019
      </Typography>
      <Typography variant='caption'>Created with love!</Typography>
    </footer>
  );
};
