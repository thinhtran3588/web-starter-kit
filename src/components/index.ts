export * from './AdminLayout';
export * from './AdminFooter';
export * from './Footer';
export * from './Header';
export * from './Icon';
export * from './Layout';
export * from './Sidebar';
export * from './SidebarMenu';
export * from './FormHeader';
export * from './Field';
export * from './FormFilter';
export * from './FormInput';
export * from './FormSearch';
export * from './Table';
export * from './LanguageSelection';
export * from './Link';
export * from './AuthLayout';
export * from './Head';
export * from './Button';
export * from './Grid';
export * from './Typography';
