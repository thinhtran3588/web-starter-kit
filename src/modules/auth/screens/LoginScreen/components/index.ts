export * from './EmailLogin';
export * from './PhoneNoLogin';
