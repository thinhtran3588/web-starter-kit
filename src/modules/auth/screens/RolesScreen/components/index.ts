export * from './Detail';
export * from './PermissionsTable';
