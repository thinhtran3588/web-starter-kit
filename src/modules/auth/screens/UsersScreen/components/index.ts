export * from './Detail';
export * from './PermissionsTable';
export * from './ChangePassword';
