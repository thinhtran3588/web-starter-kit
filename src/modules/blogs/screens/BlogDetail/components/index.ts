export * from './Content';
export * from './SideMenu';
