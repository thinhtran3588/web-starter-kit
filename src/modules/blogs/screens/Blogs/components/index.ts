export * from './Categories';
export * from './BlogsList';
