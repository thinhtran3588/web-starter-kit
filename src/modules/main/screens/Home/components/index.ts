export * from './CoverImage';
export * from './Features';
export * from './Quote';
export * from './Help';
