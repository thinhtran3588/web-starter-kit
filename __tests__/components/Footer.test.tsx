import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from '@app/components/Footer';

describe('components/Header', () => {
  beforeEach(() => {});

  it('renders successfully', async () => {
    const { baseElement } = render(<Footer />);

    expect(baseElement).toMatchSnapshot();
  });
});
