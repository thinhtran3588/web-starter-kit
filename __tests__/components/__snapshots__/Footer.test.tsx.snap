// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Header renders successfully 1`] = `
<body>
  <div>
    <footer
      class="makeStyles-root-1"
    >
      <div
        class="MuiGrid-root MuiGrid-container"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 MuiGrid-grid-lg-2"
        />
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-lg-8"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-md-3"
            >
              <h6
                class="MuiTypography-root MuiTypography-h6"
              >
                common.company
              </h6>
              <div
                class="makeStyles-linkContainer-3"
              >
                <next-link
                  href="/"
                >
                  <a
                    class="makeStyles-link-2"
                    color="inherit"
                    href="/"
                  >
                    common.home
                  </a>
                </next-link>
              </div>
              <div
                class="makeStyles-linkContainer-3"
              >
                <next-link
                  href="/aboutUs"
                >
                  <a
                    class="makeStyles-link-2"
                    color="inherit"
                    href="/aboutUs"
                  >
                    common.aboutUs
                  </a>
                </next-link>
              </div>
              <div
                class="makeStyles-linkContainer-3"
              >
                <next-link
                  href="/blogs"
                >
                  <a
                    class="makeStyles-link-2"
                    color="inherit"
                    href="/blogs"
                  >
                    common.blogs
                  </a>
                </next-link>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-6"
            >
              <h6
                class="MuiTypography-root MuiTypography-h6"
              >
                common.socialMedias
              </h6>
              <div>
                <next-link
                  href="/"
                >
                  <a
                    class="makeStyles-socialLink-4"
                    color="inherit"
                    href="/"
                  >
                    <img
                      class="makeStyles-image-5"
                      src="/static/images/facebook.png"
                    />
                  </a>
                </next-link>
                <next-link
                  href="/"
                >
                  <a
                    class="makeStyles-socialLink-4"
                    color="inherit"
                    href="/"
                  >
                    <img
                      class="makeStyles-image-5"
                      src="/static/images/linkedin.png"
                    />
                  </a>
                </next-link>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-3"
            >
              <h6
                class="MuiTypography-root MuiTypography-h6"
              >
                ©
                 
                <next-link
                  href="/"
                >
                  <a
                    class="makeStyles-link-2"
                    color="inherit"
                    href="/"
                  >
                    Thinh Tran
                  </a>
                </next-link>
                 
                2019
              </h6>
              <span
                class="MuiTypography-root MuiTypography-caption"
              >
                common.footer.createdWithLove
              </span>
            </div>
          </div>
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 MuiGrid-grid-lg-2"
        />
      </div>
    </footer>
  </div>
</body>
`;
